umm-spec-lib/src/cmr/umm_spec/validation/granule.clj

```diff
@@ -120,6 +120,12 @@
   [field-path two-d-coordinate-system]
   (let [{{min-1 :MinimumValue max-1 :MaximumValue} :Coordinate1
          {min-2 :MinimumValue max-2 :MaximumValue} :Coordinate2} (:parent two-d-coordinate-system)
+        ;; Collection v1.17.2 converted these fields to strings, convert them back to numbers if needed
+        ;; TODO - implement validation for Military Grid Reference System coordinates. Right now they are changed to nil here.
```

Comment:
Should file a ticket for this, if not done already

Reply:
Okay, we filed a ticket CMR-8796 for these TODO validation items. The TODO comments have been removed from the two files in which they appear.